code/datums/gamemodes/spy_theft.dm

```diff
@@ -721,17 +721,25 @@
 	possible_areas = get_areas_with_unblocked_turfs(/area/station)
 	possible_areas += get_areas_with_unblocked_turfs(/area/diner)
 	possible_areas -= get_areas_with_unblocked_turfs(/area/diner/tug)
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/medical/asylum)			// Donut 3 Asylum
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/security/hos)
```

Comment:
disagree - high stakes drop area doesnt mean its a problem. some spies will have the gall to drop here, others won't- thats part of choosing your target. 
Targets that are *difficult* but not otherwise obscure, unreachable or hard to understand shouldnt be removed imo.